### ISSUE TITLE
Software not registering over 24 tracks

### ISSUE THREAD

**Issue by jamilroach** [gavinbenda/platinum-md#17]
Hi, Tested on multiple machines (mzs1, n920, n910) and cannot seem to record over 24 tracks without resetting multiple times. Is this a currently known issue? Many thanks!

**Comment by gavinbenda** [comment]
Yeah there seems to be some weirdness with USB connection, this is a bit out of my control - but in a future release I will at least have the program prompt if there is an error and unplug/plug-in your device again to resume.

**Comment by steve-71** [comment]
Been doing a bit of testing with this issue and I don't think it's the number of tracks; I've got an MD with 29 tracks made with platinum-md and it was on the 29 track being successfully written that it was then unable to reload the track listing; I have another MD with 42 tracks in total; in the console log there are two separate writes where the JSON data is being split into two; I think that the stdout.on('data',..) needs to capture the output of the spawned netmdcli process and append to a buffer, then act on the content of the buffer when the spawned netmdcli process closes. This is in readNetMd function in NetMdListing.vue. 
Each time the first output of data is about the same size; just over 4k bytes; I've seen 4132 bytes, 4134 bytes, 4137 bytes; then there's a second output with the remainder. 
A workaround I've used if you need lots of tracks is to give each track short names. 
I'll have a crack at fixing it but I'm not promising anything, C# is my day job ;)
